postfix/check.py

```diff
@@ -61,7 +63,8 @@ def _get_queue_count(self, directory, queues, tags):
                 # can dd-agent user run sudo?
                 test_sudo = os.system('setsid sudo -l < /dev/null')
                 if test_sudo == 0:
-                    output, _, _ = get_subprocess_output(['sudo', 'find', queue_path, '-type', 'f'], self.log, False)
+                    postfix_user = self.init_config.get('postfix_user')
```

Comment:
without default value the new check will not work with an old configuration (ie: without the `postfix_user`). I believe that would be a breaking change (and the new version indicate a minor change). We should add a default value (`root` for backward compatibility ?)